lib/items_repo.rb

```diff
@@ -80,13 +80,18 @@ def unit_price_exists?(attributes)
 
   def update(id, attributes)
     item = find_by_id(id)
-    if item_exists?(id)
-      if unit_price_exists?(attributes)
-        item.unit_price = BigDecimal(attributes[:unit_price])
-      end
+    if item.nil?
+      nil
+      #attributes.each_key do |key|
+    elsif !attributes[:name].nil?
       item.name = attributes[:name]
+      item.updated_at = Time.now
+    elsif !attributes[:description].nil?
```

Comment:
I wonder if doing these elsifs will also prevent multiple attributes being updated at a time. Might need some helper methods here to reduce code length and make it easier to read.